tests/meta.ml

```diff
@@ -0,0 +1,67 @@
+(** Test the Meta module. *)
+open Common
+
+open Handle
+open Parsing
+open Core
+
+let term : Term.term Alcotest.testable =
+  ( module struct
+    type t = Term.term
+
+    let pp = Print.pp_term
+    let equal = Rewrite.eq
+  end )
+
+let scope_term ss t =
+  Scope.scope_term true ss [] (Term.new_problem ()) (Fun.const None)
+    (Fun.const None) t
+
+let () = Library.set_lib_root (Some "/tmp")
+
+(** The signature of [OK/bool.lp] *)
+let bool_sig = Compile.Pure.compile_file "OK/bool.lp"
+
+(** The signature state with [OK/bool.lp] opened. *)
+let bool_sig_st =
+  let open Core in
+  let ss = Sig_state.of_sign bool_sig in
+  Sig_state.open_sign ss bool_sig
+
+(** The term [bool_or true true] *)
+let or_true_true =
+  Syntax.P.(appl_list (iden "bool_or") [ iden "true"; iden "true" ])
+  |> scope_term bool_sig_st
```

Comment:
Why not using parse_string instead?

Reply:
Well we don't have a `parse_term` yet, we can only parse complete commands (unless we use https://github.com/Deducteam/lambdapi/blob/c1337fdedc14ffc4001735898f7e9796696b7a34/tests/rewriting.ml#L6)